### PR TITLE
Ldp topic data

### LINKED ISSUE #5294: Capture topic data in ATI
**Is your feature request related to a problem? Please describe.**
As a member of the audiences team

I want to be able to track LDP topic data in ATI 

so that I can understand how particular topics perform across page types 

**Describe the solution you'd like**
We need to ensure that we capture Topic data in ATI where it is added to a CPS page. 

These labels can be found in tags.about. within the ares payload for a service. 

See examples for this page: 

https://www.bbc.com/gujarati/media-51154530

https://ares-broker.api.bbci.co.uk/api/asset/gujarati/media-51154530

**Testing notes**
[This has been implemented on articles already.](https://github.com/bbc/simorgh/blob/860b7311e4f3e84926642a1536aecdaf6fbeac86/src/app/containers/ATIAnalytics/params/article/buildParams.js#L27-L28)

See https://www.bbc.co.uk/news/articles/ce9v85jlrzzo and screenshots below. 
 

We can test these tags using the ATI Inspector tool: https://marketplace.atinternet-solutions.com/product/tag-inspector/ 

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
Jira - https://jira.dev.bbc.co.uk/browse/WSMEDIA-140

Do we need to consider [metadata](https://github.com/bbc/simorgh/blob/860b7311e4f3e84926642a1536aecdaf6fbeac86/src/app/containers/ArticleMetadata/index.jsx) too?

### PR DESCRIPTION
**Pull request by jroebu14**
Resolves https://github.com/bbc/simorgh/issues/5294

**Overall change:** 

- Adds LDP thing tags to CPS ATI call.
- Refactors `ArticleMetdata` and `Metadata` components so that LDP tags in page metadata is not article specific so that CPS or other page types can render these meta tags.
- Refactors the tests and updates snapshots

**Code changes:**
- Adds `ldpThingIds` and `ldpThingLabels` in `src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js` so that these measurements are sent to ATI
- Moves meta tag rendering out of `src/app/containers/ArticleMetadata/index.jsx` and into `src/app/containers/Metadata/index.jsx` so that CPS pages (and other pages) can easily render these meta tags
- Passes `aboutTags` prop into the `Metadata` and `LinkedData` component from all CPS pages
- Refactors/fixes associated tests and updates snapshots

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

**Testing notes**

1. Go to http://localhost:7080/pidgin/media-44221514
2. Using ATI browser extension, check the correct topic data is being sent (see screenshot)

image (2) (1)

Here is an example of what to we expect to be sending:

| Field                | Condition                                  | Value                                          | Example                                                                                                                                               |
|----------------------|--------------------------------------------|------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
| LDP Things (x13)     | Include if there are any, separated by `~` | aresResponse.metadata.tags.about[i].thingLabel | `Kashmir~India~India-Pakistan relations~Pakistan`                                                                                                     |
| LDP Things IDs (x14) | Include if there are any, separated by `~` | aresResponse.metadata.tags.about[i].thingId    | `2f5b1359-19b3-47c2-92a7-70879ac4823c~5a08f030-710f-4168-acee-67294a90fc75~896b5698-8137-45c1-ba35-cf6f871fbfdd~cd832879-ac76-4842-8c33-be2be67e53b8` |

Here is an example of what you will see in the ATI dashboard (will only be able to see this in the live environment)

image (1) (1)

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]
👍

File: src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import {
   getPublishedDatetime,
   LIBRARY_VERSION,
 } from '../../../../lib/analyticsUtils';
+import { getThingAttributes } from '../../../../lib/analyticsUtils/article';
```
Comment (karinathomasbbc) [inline src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js]: Could we use #lib/analyticsUtil instead of the ../ path notation?

Comment (karinathomasbbc) [reply src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js]: Also, should this live within analyticsUtils/article, or somewhere more generic?

File: src/app/pages/CpsMap/index.jsx [hunk 2]
```diff
@@ -48,6 +48,7 @@ const CpsMapContainer = ({ pageData }) => {
   );
   const firstPublished = getFirstPublished(pageData);
   const lastPublished = getLastPublished(pageData);
+  const aboutTags = path(['metadata', 'tags', 'about'], pageData);
```
Comment (karinathomasbbc) [inline src/app/pages/CpsMap/index.jsx]: I think you can use`getAboutTags` in #lib/utilities/parseAssetData instead?

Comment (karinathomasbbc) [inline src/app/pages/CpsMap/index.jsx]: I think you can use`getAboutTags` in #lib/utilities/parseAssetData instead?

Comment (jroebu14) [reply src/app/pages/CpsMap/index.jsx]: parse in parseAssetData is maybe the wrong verb here

File: src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js (hunk 1 above)
Comment (jroebu14) [reply src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js]: I personally think things like `#lib/analyticsUtils` just hides how fragmented the code in Simorgh is

File: src/app/pages/CpsMap/index.jsx (hunk 2 above)
Comment (karinathomasbbc) [reply src/app/pages/CpsMap/index.jsx]: Yeah I think you're right. Should we add a tech_debt issue to address it later?

**Review by ryanmccombe — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me.
Screenshot 2020-01-30 at 10 40 36

**Pull request merged**